sycl/include/sycl/ext/intel/experimental/esimd/common.hpp

```diff
@@ -104,12 +104,7 @@ template <typename T> struct lsc_expand_type {
 
 template <typename T> struct lsc_bitcast_type {
 public:
-  using type = std::conditional_t<
-      sizeof(T) == 1, uint8_t,
-      std::conditional_t<
-          sizeof(T) == 2, uint16_t,
-          std::conditional_t<sizeof(T) == 4, uint32_t,
-                             std::conditional_t<sizeof(T) == 8, uint64_t, T>>>>;
+  using type = __ESIMD_DNS::lsc_bitcast_type<T>::type;
```

Comment:
i dont want to delete the experimental version in case a customer is using it, but ill get rid of the non-experimental version and define the experimental version to `uint_type_t`

Reply:
This can be removed. 
1) It is in `detail::` and must not be used by user. SYCL SPEC says: "the sycl::detail namespace are reserved for implementation details".
2) It is also in experimental:: and there is https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/ContributeToDPCPP.md#maintaining-stable-abiapi telling that we can remove it.

Reply:
ah ok good point, will remove, thanks